src/styles/blocks/_clock.scss

```diff
@@ -0,0 +1,87 @@
+.clock {
+  @include flex-center;
+  background-image: url("../images/background.jpg");
+  border: 5px solid $defaultColor;
+  height: $clockSize;
+  width: $clockSize;
+  border-radius: 50%;
+  position: relative;
+  box-shadow: inset 0 0 12px 9px rgba(0,0,0,0.75);
+
+  // Clock Elements
+  &__long-hand,
+  &__round,
+  &__short-hand {
+    position: absolute;
+    box-shadow: $defaultShadow;
+  }
+
+  &__short-hand, &__long-hand {
+    width: $handWidth;
+    background-color: $defaultColor;
+    border-radius: 20px;
+    bottom: 50%;
+    transform-origin: $defaultTransformOrigin;
+    transform: rotate(180deg);
+  }
+
+  &__short-hand {
+    height: 30%;
+    animation: rotate 100s infinite linear;
+  }
+
+  &__long-hand {
+    height: 45%;
+    animation: rotate 10s infinite linear;
```

Comment:
it will be better to change animation speed according to the task
![image](https://github.com/mate-academy/layout_stop-watch/assets/22824431/5e413bee-2b52-4b21-b53d-6557ff51338d)